masterlist.yaml

```diff
@@ -1911,6 +1911,9 @@ plugins:
       - 'Av Latta Magicka - Rebalance Complete.esp'
 
 ###### AudioVisual - Animations & Physics ######
+  - name: 'Feminized_Animations_.*\.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/47774' ]
+    tag: [ Actors.Anims ]
```

Comment:
@Infernio Looks good, but would you mind putting a newline between this entry & the next? I know that's not the case with most entries in this masterlist, but I think it'd help with readability if we did that going forward like in the newer masterlists.

Reply:
Done